ops/terraform/environments/scratch/SCRATCH-ENVIRONMENT.md

```diff
@@ -0,0 +1,17 @@
+# Terraform Scratch Environment
```

Comment:
Arguably I should pull this pseudo-environment of of here and put it in a tutorial/walkthrough directory or something. I *think* I can do that as long as the modules I depend on are somewhere I can see. I want to take another pass at this shortly.